resource.html

```diff
@@ -149,97 +149,126 @@ <h3 class="hover-underline-animation">App Development Courses <img class="course
-               src="https://img-c.udemycdn.com/course/750x422/543090_9b99_3.jpg">
+
+                        src="https://img-c.udemycdn.com/course/750x422/543090_9b99_3.jpg" alt="ruby">
+
+               <!-- src="https://img-c.udemycdn.com/course/750x422/543090_9b99_3.jpg"> -->
+
                     </a>
                     <a href="https://www.udemy.com/course/kotlin-tutorial-for-beginners/" target="_blank">
                         <img width="33%" height="300"
-               src="https://miro.medium.com/max/1200/1*8YPjY2xhNwQylBBs8dYB0g.png">
+                            src="https://miro.medium.com/max/1200/1*8YPjY2xhNwQylBBs8dYB0g.png" alt="kotlin">
                     </a>
                 </div>
-  <div class="carousel-cell">
- <a href="https://www.udemy.com/course/beg-modern-cpp/" target="_blank">
+                    <div class="carousel-cell">
+                    <a href="https://www.udemy.com/course/beg-modern-cpp/" target="_blank">
                         <img width="33%" height="300"
-                            src="https://hackernoon.com/images/VyvcKdbWHbTaN3QzRCQQS7pXASq1-9x7p315b.jpeg">
+                            src="https://hackernoon.com/images/VyvcKdbWHbTaN3QzRCQQS7pXASq1-9x7p315b.jpeg" alt="cpp">
                     </a>
 
                     <a href="https://www.udemy.com/course/learn-flutter-dart-to-build-ios-android-apps/"
                         target="_blank">
                         <img width="33%" height="300"
-                        src="https://preview.redd.it/31b2ii8hchi31.jpg?auto=webp&s=309fe75e96212cf42c4120ca5adedaef52c41e01">
+
+                            src="https://preview.redd.it/31b2ii8hchi31.jpg?auto=webp&s=309fe75e96212cf42c4120ca5adedaef52c41e01" alt="flutter">
+
+                        <!-- src="https://preview.redd.it/31b2ii8hchi31.jpg?auto=webp&s=309fe75e96212cf42c4120ca5adedaef52c41e01"> -->
+
                     </a>
-</div> 
+                    </div> 
 
                 <div class="carousel-cell">
                     <a href="https://www.udemy.com/course/javascript-essentials/" target="_blank">
                         <img width="33%" height="300"
-                            src="https://mpng.subpng.com/20180417/ite/kisspng-web-development-javascript-logo-java-plum-5ad58e75bc9174.9127947515239450777724.jpg">
+                            src="https://mpng.subpng.com/20180417/ite/kisspng-web-development-javascript-logo-java-plum-5ad58e75bc9174.9127947515239450777724.jpg" alt="web-development">
                     </a>
                     <a href="https://www.udemy.com/course/r-basics/" target="_blank">
                         <img width="33%" height="300"
-                            src="https://d1zx6djv3kb1v7.cloudfront.net/wp-content/media/2020/05/R-PROGRAMMING-LANGUAGE-i2tutorials.jpg">
+                            src="https://d1zx6djv3kb1v7.cloudfront.net/wp-content/media/2020/05/R-PROGRAMMING-LANGUAGE-i2tutorials.jpg" alt="programming language">
```

Comment:
I think this image is broken...
Since you are correcting all the images, would be better to correct this one as well :)